test/e2e/ec2/instance.go

```diff
@@ -171,3 +176,25 @@ func (c invalidInstanceProfileRetryable) IsErrorRetryable(err error) aws.Ternary
 
 	return aws.BoolTernary(false)
 }
+
+func (i *Instance) WaitForRunning(ctx context.Context, ec2Client *ec2.Client) error {
```

Comment:
is it reasonable to add these methods on the Instance struct?

Reply:
I would model it like the delete instance one
Standalone function passing the client and the instance id as arguments